packages/cli/src/cmds/updateAad.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Result, FxError, err, ok, Void, Stage } from "@microsoft/teamsfx-api";
+import path from "path";
+import yargs, { Argv } from "yargs";
+import activate from "../activate";
+import CliTelemetry, { makeEnvRelatedProperty } from "../telemetry/cliTelemetry";
+import {
+  TelemetryEvent,
+  TelemetryProperty,
+  TelemetrySuccess,
+} from "../telemetry/cliTelemetryEvents";
+import { getSystemInputs } from "../utils";
+import { YargsCommand } from "../yargsCommand";
+
+export default class UpdateAadManifest extends YargsCommand {
+  public readonly commandHead = "update aad-manifest";
```

Comment:
should the command be `update aad-app`? @jidddddd

Reply:
Will update this command to be update,
and "update aad-manifest" as the sub-command
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16271402
this command to "update teams-app-manifest" as another sub-command

Reply:
I think `update aad-manifest` is fine as that's exactly what we're doing.

Reply:
offline synced with @jidddddd let's use `update aad-app` because AAD app is an entity in AAD, not AAD manifest. Azure CLI also uses `az ad app update`.
We also need to use `update teams-app`, @nliu-ms FYI.